data/_ui/help.txt

```diff
@@ -150,6 +150,7 @@ help "shipyard"
 	`Here, you can buy new ships.`
 	`Use your scroll wheel or click and drag, to scroll the view.`
 	`As in the trading and outfitter panels, you can hold down Shift, Control, or Alt to buy 5, 20, or 500 ships at once, or multiple keys for larger amounts.`
+	`Clicking a ship while holding Alt will lock it, preventing it from being sold. Alt+clicking will unlock it again.`
```

Comment:
```suggestion
	`Clicking a ship while holding the Alt key will lock it, preventing it from being sold. Alt+clicking will unlock it again.`
```

I'd rather say it like that personally.

Reply:
"Clicking a ship while holding the Alt key will lock it, preventing it from being sold. Alt+clicking a second time will unlock it .` 

IMO is even clearer.

Reply:
"IMO"?

Reply:
in my opinion